lib/radiator_support/generator/user_agents_downloader.ex

```diff
@@ -0,0 +1,25 @@
+defmodule RadiatorSupport.UserAgentsDownloader do
+  @moduledoc """
+  Downloads file with 10.000 user agents for RadiatorSupport.TrackingGenerator
+  because keeping a 1MB file with noncritical data versioned here feels wrong.
```

Comment:
nah - just put it in. git zips resources anyways. And having this weird extra dependenchy on a gist isn't better imho.